src/LanguageServer/Protocol/Protocol/Internal/VSInternalReferenceItem.cs

```diff
@@ -31,7 +31,7 @@ public int Id
         /// Gets or sets the reference location.
         /// </summary>
         [JsonPropertyName("_vs_location")]
-        public Location Location
+        public Location? Location
```

Comment:
This and other changes involving `Location` were made to explicitly show that it can be null, and gracefully handle that case when using the finder via the LSP